.github/workflows/integrate.yml

```diff
@@ -47,9 +48,11 @@ jobs:
           python3 -m pip install coverage
           python3 -m pip install opencv-python
       - name: Run tests
+        working-directory: ./src
         run: |
-          python3 -m robotpy test
-      # Action to preview HTML files currenctly not working: https://github.com/pavi2410/html-preview-action
+          python3 -m unittest
```

Comment:
Does this run the adequate robotpy tests? 

It's not a big deal if it doesn't, but it isn't very useful if it isn't. Probably better to just remove all testing from the workflow.

Reply:
It will do discovery and run unittests it finds. We can't run any robotpy tests it appears so whether to run anything is up to us.